### PR TITLE
:octocat: Capstone project submission

### PR DESCRIPTION
**Pull request by mahabubx7**
### :octocat: Capstone project submission

:heavy_check_mark: Added contents and implemented in UI
:heavy_check_mark: Added Mobile-first design
:heavy_check_mark: Added Desktop design
:heavy_check_mark: Coded Mobile-menu and more button
:heavy_check_mark: Coded dynamic `speakers` section with JavaScript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,169 @@
...
+
+- **[Clean and Simple Static Webpages]**
+- **[Only Raw HTML-CSS/SCSS-JS codes]**
+- **[Nodejs for development tools]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (Shedrack-Sunday) [inline README.md:70]: - [ ] Nice on deploying the project. However, based on project requirements, you're meant to also record a 5-minute video describing the keynote points of your project. This is in line with having a well-documented code base. Kindly check the image below for detailed instructions. 
- You can use [Loom](https://www.loom.com/looms/videos) for this

## Instructions:
image

File: README.md [hunk 2]
```diff
@@ -1 +1,169 @@
...
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```
Comment (Shedrack-Sunday) [inline README.md:27]: - [x] Seems you forgot this section of your README. Kindly add 1 to 3 future features that you might possibly add to the project. This was placed with best practices in mind for good documnetion of your project 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,169 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the `microverse` team for this nice ui (behance-shot).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
```
Comment (Shedrack-Sunday) [inline README.md]: - [x] The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Kindly add it to the README of your project.

## Instructions:
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    <nav id="topbar">
+      <div class="container">
+        <ul class="menu">
+          <li><a href="#"><i class="ri-facebook-circle-line"></i></a></li>
+          <li><a href="#"><i class="ri-twitter-line"></i></a></li>
+          <li><a href="#"><i class="ri-english-input"></i></a></li>
+          <li><a href="#">my page</a></li>
+          <li><a href="#">logout</a></li>
+        </ul>
+      </div>
```
Comment (Shedrack-Sunday) [inline about.html:30]: - [x] You did well implementing this feature, however, it's meant to have a dark background color as provided in the design guide. This gives a better UX and feels to the project. Kindly fix this 👍 

## Design given:
image

## Your design:
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+            <p class="txt-tiny">Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam reiciendis recusandae nobis suscipit quo nemo aliquam perspiciatis voluptatem aspernatur aliquid?</p>
+          </div>
+        </div>
+      </div>
+    </footer> <!--//Footer-->
+  </main> <!--//Body Content-->
+
+  <!--JavaScripts-->
+  <script src="./script.js"></script>
+</body>
```
Comment (Shedrack-Sunday) [inline about.html]: - [x] You did well sticking to the same footer on both the `home` and `about` sections 👏 . However, there's a few adjustments you need to make:
- The about has a black background on the footer
- The partner section is not present 

- [x] I've made a detailed image comparison for you to view and make the appropriate change. Good job so far 👍 

## Design:
image

## Your implementaion:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @mahabubx7       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, You still need to work on a few stying, video presentation, and a couple more which I highlighted below 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by amejid — APPROVED** [review]
Hi @mahabubx7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Great presentation :heavy_check_mark: 
- Website is responsive :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**